tools/node_wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+export PATH=$(echo $PATH | cut -d : -f 2-) # Do not call oneself recursively
```

Comment:
This is a bit fragile. There could be other directories added by dune. I'll commit an alternative.

Reply:
Let's keep it this way for now.